cmd/create/cluster/cmd.go

```diff
@@ -351,6 +353,14 @@ func initFlags(cmd *cobra.Command) {
 	)
 	flags.MarkHidden(ClassicOidcConfigFlag)
 
+	flags.BoolVar(
+		&args.externalAuthProvidersEnabled,
+		ExternalAuthProvidersEnabledFlag,
+		false,
+		"Enable external auth configuration of a Hosted Control Planes.",
```

Comment:
Maybe not mix singular and plural?

"Enable external auth configuration for a Hosted Control Plane cluster."

Reply:
Thanks, will address that